src/tests/system/tests/test_passkey.py

```diff
@@ -241,3 +241,188 @@ def test_passkey__su_srv_not_resolvable(
         ioctl=f"{moduledatadir}/umockdev.ioctl",
         script=f"{testdatadir}/umockdev.script.{suffix}",
     )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__offline_su(client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str):
+    """
+    :title: Check offline su authentication of a user with LDAP, IPA, AD and Samba
+    :setup:
+        1. Add a LDAP, IPA, AD and Samba user with passkey_mapping.
+        2. In SSSD domain change "cache_credentials" and "krb5_store_password_if_offline" to "True"
+        3. In SSSD pam change "offline_credentials_expiration" to "0"
+        4. Setup SSSD client with FIDO and umockdev, start SSSD service.
+    :steps:
+        1. Check su authentication of the user.
+        2. Make server offline by removing rules for LDAP, KDC and Global Catalog ports.
```

Comment:
You should mention that these are firewall rules. In addition, I think you are creating those rules, not removing them.

Reply:
Yes, updated the comment.

Reply:
Looks good